modules/ROOT/pages/sysconfig-network-configuration.adoc

```diff
@@ -890,3 +890,43 @@ storage:
           master=bond0
           slave-type=bond
 ----
+=== Disabling Automatic Configuration on Ethernet Devices (DHCP/SLAAT)
+
+==== via Butane
+
+To disable automatic configuration on ethernet devices (DHCP/SLAAT) with no other matching connections, add this to your Butane file.
+
+[source, yaml]
+----
+storage:
+  files:
+    - path: /etc/NetworkManager/conf.d/noauto.conf
+      mode: 0644 
+      contents: 
+        inline: |
+          [main]
+          # Do not do automatic (DHCP/SLAAC) configuration on ethernet devices
+          # with no other matching connections.
+          no-auto-default=*
+----
+
+Adding `no-auto-default=*` to a config file in /etc/NetworkManager/conf.d/ disables automatic configuration of ethernet devices.
+
+==== Butane example
+
+.Example
+[source, yaml]
+----
+variant: fcos
+version: 1.4.0
+storage:
+  files:
+     - path: /etc/NetworkManager/conf.d/noauto.conf
+      mode: 0644 
+      contents: 
+        inline: |
+          [main]
+          # Do not do automatic (DHCP/SLAAC) configuration on ethernet devices
+          # with no other matching connections.
+          no-auto-default=* 
+----
```

Comment:
Let's add a `NOTE:` below here to let people know if they disable autoconfiguration but don't provide configuration for any NIC then their system won't be connected to a network at all.

```
WARNING: If NetworkManager autoconfiguration of ethernet devices is disabled and no other network configuration is provided the system will boot without network access.
```